### PR TITLE
Doc app layer tutorial updates

### PR DESCRIPTION
**Pull request by bmckercher123**
Updates from testing by Alina during onboarding. 

Alina has approved/reviewed the changes to steps.

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 73d8ccc2f0a0316d7fadad8c824fbacac247f3f4

https://deploy-preview-2669--calico.netlify.com

File: master/security/app-layer-policy/index.md [hunk 1]
```diff
@@ -3,22 +3,23 @@ title: Application layer policy tutorial
...
 
-To create a Kubernetes cluster which supports application layer policy, follow
-one of our [getting started guides]({{site.baseurl}}/{{page.version}}/getting-started/),
-and [enable application layer policy]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/app-layer-policy).
-
-You will also need the [calicoctl]({{site.baseurl}}/{{page.version}}/getting-started/calicoctl/install) command line tool.
+1. Build a Kubernetes cluster.
+1. Install Calico on Kubernetes:  
+- If Calico is not installed on Kubernetes, see [Calico on Kubernetes]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes).  
+- If Calico is already installed on Kubernetes, verify that you have: Calico networking with [Kubnernetes Data Store over etcd (recommended)]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/calico) or a non-Calico CNI, and Calico network policy.
```
Comment (caseydavenport) [inline master/security/app-layer-policy/index.md]: I don't think the "recommended" bit here is right - I don't think we would say etcd is recommended over other installation approaches.

Comment (caseydavenport) [reply master/security/app-layer-policy/index.md]: Also, I don't think this tutorial requires that etcd be in use

File: master/security/app-layer-policy/index.md [hunk 2]
```diff
@@ -3,22 +3,23 @@ title: Application layer policy tutorial
...
-To create a Kubernetes cluster which supports application layer policy, follow
-one of our [getting started guides]({{site.baseurl}}/{{page.version}}/getting-started/),
-and [enable application layer policy]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/app-layer-policy).
-
-You will also need the [calicoctl]({{site.baseurl}}/{{page.version}}/getting-started/calicoctl/install) command line tool.
+1. Build a Kubernetes cluster.
+1. Install Calico on Kubernetes:  
+- If Calico is not installed on Kubernetes, see [Calico on Kubernetes]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes).  
+- If Calico is already installed on Kubernetes, verify that you have: Calico networking with [Kubnernetes Data Store over etcd (recommended)]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/calico) or a non-Calico CNI, and Calico network policy. 
+1. Install [calicoctl command line tool]({{site.baseurl}}/{{page.version}}/getting-started/calicoctl/install).
```
Comment (caseydavenport) [inline master/security/app-layer-policy/index.md]: Install _the_ calicoctl command line tool

File: master/security/app-layer-policy/index.md (hunk 1 above)
Comment (bmckercher123) [reply master/security/app-layer-policy/index.md]: Spike confirmed that you are correct. Either works. Fixed text.

File: master/security/app-layer-policy/index.md (hunk 2 above)
Comment (bmckercher123) [reply master/security/app-layer-policy/index.md]: Fixed.

File: master/security/app-layer-policy/index.md [hunk 3]
```diff
@@ -144,7 +138,7 @@ Notice that from here, we get direct access to the backend database.  For exampl
 
 (Piping to `python -m json.tool` nicely formats the output.)
 
-#### Single factor authorization
+#### Single-factor authentication
```
Comment (caseydavenport) [inline master/security/app-layer-policy/index.md]: There's another instance of this here: https://github.com/projectcalico/calico/pull/2669/files#diff-8b697d196bb8d59ad589f28c5efd64efR289

Comment (bmckercher123) [reply master/security/app-layer-policy/index.md]: Fixed.

**Pull request merged**